docs/architecture/serverless/serverless-business-scenarios.md

```diff
@@ -27,7 +27,7 @@ The full repository and instructions to build the logic app are available here:
 
 See how to implement a simple serverless Azure Function in the Azure Web Portal or in Visual Studio. Build a client with Xamarin.Forms that runs on Android, iOS, and Windows. The application is then refined to use JavaScript Object Notation (JSON) as a communication medium between the server and the mobile clients with a serverless back end.
 
-For more information, see [Implementing a simple Azure Function with a Xamarin.Forms client](https://azure.microsoft.com/resources/samples/functions-xamarin-getting-started/).
+For more information, see [Implementing a simple Azure Function with a Xamarin.Forms client](https://docs.microsoft.com/samples/azure-samples/functions-xamarin-getting-started/implementing-a-simple-azure-function-with-a-xamarinforms-client/).
```

Comment:
The old link was working for this one, but it redirects to the samples browser. So, I preferred to include the samples browser link directly.

Reply:
Yep, always better to avoid redirection.